extension_jwt.go

```diff
@@ -0,0 +1,155 @@
+package helix
+
+import (
+	"encoding/base64"
+	"fmt"
+	"time"
+
+	"github.com/dgrijalva/jwt-go"
```

Comment:
Does the Go standard library have equivalent support for this package? This package hasn't been updated in years and is no longer maintained.

Additionally, I was hoping that we would not have to use any external packages in this project.

Reply:
I understand that this may not be the best package out there, I just used this module as a reference
as Twitch example projects in GO utilized it and does the job perfectly, even in my own available go module.

I think it'll be mega overkill to re-invent the wheel in all honesty...
is this something you'll budge on?

Reply:
perhaps switching to https://github.com/golang-jwt/jwt directly

Reply:
I'd rather get these endpoints supported and then i'd be happy to "in-house" the dependencies if you so wish

Reply:
I'm happy to budge. Give me a few days to review. It's a busy time for me atm.

Reply:
hey @nicklaw5 i understand entirely!

This PR is still a [W]ork [I]n [P]rogress, but I will continue to add on to it over the next week or so